.gitignore

```diff
@@ -1,3 +1,11 @@
-node_modules
-bower_components
-dist
+node_modules/
+dist/
+.idea/
+
+# only for testing, these resources should not be delivered but are easy to forget
+webapp/resources
```

Comment:
ignore whole webapp folder?

Reply:
if we ignore the whole webapp folder we won't have anything published to git